src/app/app.module.ts

```diff
@@ -19,6 +19,7 @@ import { PasswordResetComponent } from './pages/password-reset/password-reset.co
 import { EnterResetPasswordComponent } from './pages/enter-reset-password/enter-reset-password.component';
 import { AuthGuard } from './guards/auth.guard';
 import { NoAuthGuard } from './guards/no-auth.guard';
+import { DepositsComponent } from './pages/deposits/deposits.component';
```

Comment:
Hey @Genza999 , we should use absolute imports across the app as discussed earlier.

Reply:
We may also want to remove this line as well as it looks like we don't use it.

Reply:
Thanks @CryceTruly i have worked on this!